rootston/rootston.ini.example

```diff
@@ -29,7 +29,7 @@ map-to-output = VGA-1
 # Restrict cursor movements for this mouse to concrete rectangle
 geometry = 2500x800
 
-[keyboard]
+[keyboard:*]
```

Comment:
I would prefer to use `keyboard:default` instead of `keyboard:*`, because the latter feels like I'm setting sth for all keyboard, not just that this will be the default configuration (and all keyboard that don't have a specific config will fallback to the default).

Reply:
I would just call this section `[keyboard]`